### ISSUE TITLE
[Question] 'Emergency' stop? And disable interaction?

### ISSUE THREAD

**Issue by bencresty** [davidfig/pixi-viewport#374]
Hi, sorry for opening so many issues here lately, but I have one question still;
when animating the viewport from own code with gsap I want to break all currently running interactions and animations when that's starting to prevent conflicts. I did a walk through the methods, but can't find such a thing.  so...

1) How can we break all interactions and animations?
2) Is there a way to (temporary) disable user interactions?

It would be very nice if that would be possible! Thanks in advance!

**Comment by davidfig** [comment]
Viewport.pause = true should do it.

**Comment by bencresty** [comment]
That looks great, but when setting pause to false again it's continuing the animation. What I'm looking for is a complete cancel. Is that possible?

**Comment by davidfig** [comment]
I don't think we built logic to cancel the animations. You'll have to do that manually. (Feel free to submit a PR if you can generalize it.)
